tests/core/test_model.py

```diff
@@ -194,14 +193,22 @@ def test_model_validation_union_query():
         model.validate_definition()
 
 
-def test_partitioned_by():
-    expressions = parse(
-        """
+@pytest.mark.parametrize(
+    "partition_by_input, partition_by_output",
+    [
+        ("a", ["a"]),
+        ("(a, b)", ["a", "b"]),
+        ("TIMESTAMP_TRUNC(a, DAY)", ["TIMESTAMP_TRUNC(a, DAY)"]),
```

Comment:
Do we have a test for expressions that don't reference a column?

Reply:
No, but we should. Good catch.